src/troute-nwm/src/nwm_routing/__main__.py

```diff
@@ -28,6 +33,689 @@
 
 LOG = logging.getLogger('')
 
+'''
```

Comment:
we really don't need to maintain v02 main, as long as the functionality of v04 main works as needed.  I would propose dropping v02 main and make sure v04 (which should probably just be `main`) is functional.